pfio/v2/s3.py

```diff
@@ -380,7 +423,8 @@ def _reset(self):
 
     def _connect(self):
         # print('boto3.client options:', kwargs)
-        self.client = boto3.client('s3', **self.kwargs)
+        obj = boto3.client('s3', **self.kwargs)
+        self.client = Boto3ProfileWrapper(obj)
```

Comment:
It's better to switch here between the vanilla boto3 and the wrapped boto3 client by `self.trace` as well as `Local` fs does.
https://github.com/pfnet/pfio/blob/9bf5900d1aa8403633eb9a15af957aa11ff80a19/pfio/v2/local.py#L123-L127

Reply:
Forgot to change from test code ...

Reply:
fix and rebase `master` branch